server/src/main/java/org/apache/uniffle/server/ShuffleServerGrpcService.java

```diff
@@ -186,6 +186,11 @@ public void sendShuffleData(SendShuffleDataRequest req,
     String appId = req.getAppId();
     int shuffleId = req.getShuffleId();
     long requireBufferId = req.getRequireBufferId();
+    long sendTime = req.getSendTime();
+    if (sendTime > 0) {
+      shuffleServer.getGrpcMetrics().recordSendTime(ShuffleServerGrpcMetrics.SEND_SHUFFLE_DATA_METHOD,
```

Comment:
Do we need consider the data size when we calculate the metrics?

Reply:
Make sense. Could we add some comments to explain why we don't choose to use the size of data?